packages/conform-react/hooks.ts

```diff
@@ -61,6 +61,14 @@ export function useForm<
 			 * Default to `true`.
 			 */
 			defaultNoValidate?: boolean;
+
+			/**
+			 * Define if the input could be updated by conform.
+			 * Default to inputs that are configured using the `getInputProps`, `getSelectProps` or `getTextareaProps` helpers.
+			 */
+			shouldManageInput?: (
+				element: HTMLInputElement | HTMLSelectElement | HTMLTextAreaElement,
+			) => boolean;
```

Comment:
don't forget buttons! :)

Reply:
Conform does not manage the buttons (i.e. It should not update the button value). Is there something you want to achieve with this option?

Reply:
ah, okay, I was thinking user intent buttons (not conform intent buttons) but if thats outside this scope, please ignore my comment.